package.json

```diff
@@ -54,6 +54,9 @@
     "wait-on": "7.2.0"
   },
   "pnpm": {
+    "overrides": {
+      "eslint-config-galex>typescript": "$typescript"
+    },
```

Comment:
Please ignore this, it will be removed once I merge #1659 and restore `moduleResolution: "bundler"`.

Reply:
Okay :smile: